Doc/c-api/list.rst

```diff
@@ -38,9 +38,14 @@ List Objects
    .. note::
 
       If *len* is greater than zero, the returned list object's items are
-      set to ``NULL``.  Thus you cannot use abstract API functions such as
-      :c:func:`PySequence_SetItem`  or expose the object to Python code before
-      setting all items to a real object with :c:func:`PyList_SetItem`.
+      set to ``NULL``. Thus you cannot use abstract API functions such as
+      :c:func:`PySequence_SetItem` or expose the object to Python code before
+      setting all items to a real object with :c:func:`PyList_SetItem` or
+      :c:func:`PyList_SET_ITEM()`. The following APIs are safe APIs before
+      the list is fully initialized.
+      - :c:func:`PyList_SetItem()`
+      - :c:func:`PyList_SET_ITEM()`
```

Comment:
The list is not rendered properly. Maybe just write: "The following ...: PyList_SetItem and PyList_SET_ITEM."

Reply:
Rendered doc: https://cpython-previews--121626.org.readthedocs.build/en/121626/c-api/list.html